docs/source/buildingsync.rst

```diff
@@ -33,7 +33,7 @@ This process should be followed in the future when a new version of BuildingSync
 
 10. **BETTER** (seed/analysis_pipelines/better/buildingsync.py) - this pulls the buildingsync version from the settings file. Updating the version should just work, but test anyway and let BETTER team know so they can test/adjust the version on their end too. We want SEED-AT-BETTER all coordinated.
 
-11. **BSYNCR Server** - don’t touch it. This functionality uses its own version of buildingsync right now  (v2.2.0) and will need to be updated separately one day!
+11. **BSYNCR Server** - Update bsyncr-server to latest version of BuildingSync. Then update seed/analysis_pipelines/bsyncr.py to v2.6.0.
```

Comment:
I think the step is to "update the building sync files in the bsyncr tests to v2.6.0 and run the tests". There is no explicit version checks happening, from what I can tell.

Reply:
thanks, I updated!